sqlx_user_crud/src/controller/group_controller.rs

```diff
@@ -0,0 +1,81 @@
+use actix_web::{get, post, patch, delete, web, Responder, HttpResponse};
+use serde::{Deserialize, Serialize};
+use super::AppState;
+use super::log_request;
+
+pub fn init(cfg: &mut web::ServiceConfig) {
+    cfg.service(get_group_by_id);
+    cfg.service(post_group);
+    cfg.service(patch_group_by_name);
+    cfg.service(delete_group_by_name);
+}
+
+// TODO: provide response headers
```

Comment:
you probably should tend to all these pending `TODO`s

Reply:
This typo has been resolved.